test/libyul/Parser.cpp

```diff
@@ -511,6 +517,9 @@ BOOST_AUTO_TEST_CASE(customSourceLocations_invalid_suffix)
 	EVMDialectTyped const& dialect = EVMDialectTyped::instance(EVMVersion{});
 	shared_ptr<Block> result = parse(sourceText, dialect, reporter);
 	BOOST_REQUIRE(!!result);
+	BOOST_REQUIRE(errorList.size() == 1);
+	BOOST_TEST(errorList[0]->type() == Error::Type::SyntaxError);
+	BOOST_TEST(errorList[0]->errorId() == 8387_error);
```

Comment:
`/// @src 0:420:680foo` used to result in the tag being ignored. Now it's a syntax error.